### ISSUE TITLE
NA values

### ISSUE THREAD

**Issue by abarbour** [kwstat/lucid#1]
It would be quite useful to be able to replace `NA` values with something (e.g., blank space) _after_ the formatting stage.  Something along the lines of:

```
lucid(X, na.replace=" ")
```

defaulting to 

```
lucid(X, na.replace=NULL)
```

where `is.null(na.replace)` indicates no replacement.  Perhaps nested `[ls]apply` and `gsub`?

```
data.frame(apply(X, 2, function(x) apply(matrix(x), 1, function(y) gsub('NA', "--", y))))
```

**Comment by kwstat** [comment]
Released version 1.1 to CRAN.  Look for it in a few days.

**Comment by abarbour** [comment]
Great, thanks!
